src/timetable/timetable-navigator/timetable-navigator.style.tsx

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from "styled-components";
+import { Color } from "../../color";
+import { IconButton } from "../../icon-button";
+import { Text } from "../../text";
+
+export const StyledRefreshButton = styled(IconButton)<{ $isLoading: boolean }>`
+    color: ${Color.Neutral[3]};
+    @keyframes spin {
```

Comment:
to use styled components keyframe to ensure the name is unique https://styled-components.com/docs/basics#animations